### DISCUSSION TITLE
Error while activating extension

### DISCUSSION THREAD

**Discussion by cyco77** [Power-Maverick/DataverseDevTools-VSCode#146] [General]
Hi,
I was a little in stress the last couple of month... hope to work on my tasks in the next couple of weeks :)

But now I have an issue on my Mac M1:

While activation of the extension I get an error an nothing works as normal:

Activating extension 'danish-naglekar.dataverse-devtools' failed: dlopen(/Users/larshildebrandt/.vscode/extensions/danish-naglekar.dataverse-devtools-2.1.10/node_modules/keytar/build/Release/keytar.node, 0x0001): tried: '/Users/larshildebrandt/.vscode/extensions/danish-naglekar.dataverse-devtools-2.1.10/node_modules/keytar/build/Release/keytar.node' (not a mach-o file), '/System/Volumes/Preboot/Cryptexes/OS/Users/larshildebrandt/.vscode/extensions/danish-naglekar.dataverse-devtools-2.1.10/node_modules/keytar/build/Release/keytar.node' (no such file), '/Users/larshildebrandt/.vscode/extensions/danish-naglekar.dataverse-devtools-2.1.10/node_modules/keytar/build/Release/keytar.node' (not a mach-o file).

To be honest I have no idea what'S going on there....

When I start the latest version from code (debugging the extension) everything works...

May be you have an idea

greets
Lars

**Comment by cyco77** [comment]
right now you build the extension with an old version of node:
![image](https://github.com/Power-Maverick/DataverseDevTools-VSCode/assets/1198698/a9b3643f-4acb-494a-9987-0f3f86c7c2aa)
maybe you can upgrade you build action?

Thanks
Lars

**Comment by Power-Maverick** [comment]
Thank you I am looking into it as well

**Comment by Power-Maverick** [comment]
This is now resolved with v2.2.0
